src/js/mep-feature-universalgoogleanalytics.js

```diff
@@ -8,8 +8,8 @@
 $.extend(mejs.MepDefaults, {
 	googleAnalyticsTitle: '',
 	googleAnalyticsCategory: 'Videos',
-	googleAnalyticsEventPlay: 'Play',
-	googleAnalyticsEventPause: 'Pause',
+	googleAnalyticsEventPlay: 'mejs.play',
```

Comment:
I don't know if this feature really requires translation, given that not all of the elements listed are translated. Thoughts?